terminal.js

```diff
@@ -166,30 +181,26 @@ class Terminal {
 
         const root = this._getRootByPath(dirName);
         try {
-            this.fileSystem.createFile(dirName, root);
+            this.fileSystem.createRegularFile(dirName, root);
         } catch(error) {
             console.log(error.message);
         }
     }
 
     execute(commandInput) {
         const [commandName, parameter] = commandInput.split(' ');
```

Comment:
this maybe have problems, because you obtain only the first parameter from a command.
(e.g. `mkdir folder1 folder2` only creates 1 folder)
use spread operator to get all parameters. 
```
const [commandName, ...allParameters] = commandInput.split(' ');
```